resources/assets/js/stories/H2Components/Dialog.stories.tsx

```diff
@@ -22,28 +22,26 @@ stories.add(
         <p>{text("Trigger", "Open Dialog", "Text")}</p>
       </Dialog.Trigger>
       <Dialog id="dialog">
-        <Dialog.Header
-          data-h2-grid="b(middle, contained, padded, .5)"
-          className="gradient-left-right"
-        >
+        <Dialog.Header className="gradient-left-right">
           <Dialog.Title
             data-h2-padding="b(all, .5) b(left, 1)"
             data-h2-font-color="b(white)"
             data-h2-font-size="b(h4)"
-            data-h2-grid-item="b(1of2)"
           >
-            Dialog Title
+            Dialog Title but what if this were much longer?
```

Comment:
Ya I noticed that the default dialog exit trigger and title, would overlap if the title is very long, or if viewed on a mobile screen. That's why I used the grid property to ensure the text would break before overlapping.

Reply:
According to Josh its a bug with H2 which he can fix soon